tests/python_tests/io_test.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from ljmd import *
+
+class IOtest(unittest.TestCase):
+	def test_success(self):
+		assert(5==5)
```

Comment:
Why this lines?

Reply:
@lleon95 This is outdated. Please check the latest version